### PR TITLE
Add OpenLiberty springboot app

### PR DESCRIPTION
**Pull request by jandro996**
## Description

 - Add weblog-variant: spring-boot-openliberty
 - Fix CryptoExamples doCypher method
 - Marked as bug:
 
> tests/test_telemetry.py::Test_Telemetry::test_app_started_sent_only_once
> tests/test_telemetry.py::Test_Telemetry::test_seq_id
> tests/appsec/test_traces.py::Test_AppSecEventSpanTags::test_header_collection
> tests/appsec/waf/test_addresses.py::Test_BodyUrlEncoded::test_body_value
> tests/appsec/waf/test_addresses.py::Test_ResponseStatus::test_basic
> tests/appsec/waf/test_rules.py::Test_DiscoveryScan::test_security_scan
>tests.appsec.waf.test_miscs.Test_404.test_404
>tests.appsec.test_reports.Test_StatusCode.test_basic

## Check list

Your PR is not ready to be reviewed? Please save it as a draft :pray:

- [x] Follows the style guidelines of this project (See [how to easily lint the code](https://github.com/DataDog/system-tests/blob/main/docs/edit/lint.md))
- [x] CI is passing
- [x] There is at least one approval from code owners

Yes to all? Feel free to merge it whenever you want :heart:

### CODE REVIEW

File: tests/appsec/iast/test_iast.py [hunk 1]
```diff
@@ -21,10 +21,13 @@ class Test_Iast(BaseTestCase):
 
     if context.library == "nodejs":
         EXPECTED_LOCATION = "/usr/app/app.js"
+        WEAK_CIPHER_ALGORITHM = "des-ede-cbc"
```
Comment (smola) [inline tests/appsec/iast/test_iast.py:24]: I think this is from a different PR?

Comment (jandro996) [reply tests/appsec/iast/test_iast.py:24]: yes, this PR is for testing at this moment, I have to rebase the branch

File: tests/appsec/test_reports.py [hunk 2]
```diff
@@ -36,6 +36,11 @@
...
     """ Appsec reports good status code """
 
+    @irrelevant(
+        library="java",
+        weblog_variant="spring-boot-openliberty",
+        reason="This is a bug but @bug annotation fails, https://datadoghq.atlassian.net/browse/APPSEC-6583",
+    )
     def test_basic(self):
         r = self.weblog_get("/path_that_doesn't_exists/", headers={"User-Agent": "Arachni/v1"})
         assert r.status_code == 404
```
Comment (cbeauchesne) [inline tests/appsec/test_reports.py:46]: can you retry with the bug decorator,  replace this line with, and retry ? 

```suggestion
        interfaces.library.add_assertion(r.status_code == 404)
```

**Review by cbeauchesne — CHANGES_REQUESTED** [review]

File: tests/appsec/test_reports.py (hunk 2 above)
Comment (cbeauchesne) [reply tests/appsec/test_reports.py]: you did 

```
assert  interfaces.library.add_assertion(r.status_code == 404)
```

where it should be

```
  interfaces.library.add_assertion(r.status_code == 404)
```

;)

Comment (jandro996) [reply tests/appsec/test_reports.py]: :scream: I'm so sorry

**Comment by cbeauchesne** [comment]
don't mind about the performance test, it's not related

**Comment by jandro996** [comment]
> don't mind about the performance test, it's not related

Ok! so only the approval for the merge would be missing

**Review by cbeauchesne — APPROVED** [review]

**Pull request merged**